src/js/controller/action.js

```diff
@@ -1,16 +1,24 @@
+import RenderList from './render-list';
+import { AddtoStorage } from './storage';
+
 export const ArrangeList = (list) => list.map((item, index) => ({ ...item, index }));
 
 export const AddtoList = (description, list) => {
-  const todo = [{ index: 1, description, completed: false }, ...list];
+  const todo = [...list, { index: 1, description, completed: false }];
   return ArrangeList(todo);
 };
 
 export const setCompleted = (id, value, list) => {
   const todos = list.map((todo) => {
-    if (todo.index === id) { return { ...todo, completed: value }; }
+    if (todo.index.toString() === id) { return { ...todo, completed: value }; }
     return todo;
   });
   return todos;
 };
 
-export const removeCompleted = (list) => list.filter((todo) => !todo.completed);
\ No newline at end of file
+export const removeCompleted = (list) => {
+  const todos = ArrangeList(list.filter((todo) => !todo.completed));
+  AddtoStorage(todos);
+  RenderList(todos);
+  return todos;
+};
```

Comment:
- [ ] according to the requirements the user should have the possibility to remove 1 single list item independently, right now it is only possible to remove the list item if it is marked as completed by using the butter remove completed. 
- [ ] there is also no way for the user to edit the existing note, please create the possibility for them to do so.